lib/IR/IR.cpp

```diff
@@ -428,6 +428,28 @@ static void dumpIRInContext(const Value *V, llvm::raw_ostream &out) {
   }
 }
 
+std::list<const Constant *> IRFunction::getConstants() const {
```

Comment:
Do we want to call this `findConstants()` too, since it's also not just a simpler getter? Same for `getPlaceholders()`.